client/state/index.js

```diff
@@ -111,9 +115,16 @@ export const reducer = combineReducers( {
 	ui,
 	users,
 	wordads,
-} );
+};
 
-export function createReduxStore( initialState = {} ) {
+/**
+ * Create a default set of middleware/enhancers.
+ *
+ * The enhancers included depend on the runtime environment of this code.
+ *
+ * @return { array } The array of enhancers to be configured in a redux store.
+ */
+export function createEnhancers() {
```

Comment:
okay. that doesn't match what I think of in `create` but we're allowed to disagree.

this function doesn't generate any enhancer functions whose behavior is dependent on runtime circumstances. it merely returns a list of enhancers which are generated in other files…